datasetinsights/estimators/faster_rcnn.py

```diff
@@ -192,12 +197,23 @@ def train(self, train_data, val_data=None, **kwargs):
         val_loader = dataloader_creator(
             config, val_dataset, val_sampler, VAL, self.distributed
         )
-        self.train_loop(
-            train_dataloader=train_loader,
-            label_mappings=label_mappings,
-            val_dataloader=val_loader,
-            train_sampler=train_sampler,
-        )
+
+        params = self.config.clone()
+        if params.get(TrackerFactory.TRACKER):
+            params.pop(TrackerFactory.TRACKER)
```

Comment:
*fixme*
what's the point of poping this key from the dictionary? it may result in some function logging a config that is different than the one actually used

Reply:
Poping from clone of config so that only hyperparameter details can be logged to mlflow.